proto/regen/bond/v1alpha1/tx.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+package regen.bond.v1alpha1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/bond";
+
+// Msg defines the Msg service.
+service Msg {
+
+  // IssueBond issues a new bond
+  rpc IssueBond(MsgIssueBond) returns (MsgIssueBondResponse);
+
+  // SellBond sells a bond to a new owner
+  rpc SellBond(MsgSellBond) returns (MsgSellBondResponse);
+}
+
+// MsgIssueBond is the Msg/IssueBond request type.
+message MsgIssueBond {
+
+  // emission_denom is the unique id of emission
+  string emission_denom = 1;
```

Comment:
how about calling it `bond_denom` instead introducing new entity (emission)?

Reply:
or maybe  `bond_id`? .... but wait... maybe it's worth to consider it as an NFT? Let's move that discussion to a  discussion: https://github.com/regen-network/regen-ledger/discussions/815